### PR TITLE
Feature: add 'new bid' card

### LINKED ISSUE #162: Small bid contract card

### PR DESCRIPTION
**Pull request by rpollock03**
Resolves #162 - adds the new bid card. Mostly front end because some related components are not completed, therefore a separate task will be to connect these together and with backend.

Screenshot 2023-01-08 at 09 23 15

### CODE REVIEW

File: components/NewBid.tsx [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+
+const Heading = styled(Typography)({
+  color: "#101828",
+  fontSize: "1.125rem",
+  lineHeight: "1.75rem",
+  fontWeight: 500,
+})
+
+const Buttonselect = styled(ToggleButton)`
+textDecoration: 'none
```
Comment (piotrostr) [inline components/NewBid.tsx:32]: stranded `'`

File: components/NewBid.tsx [hunk 2]
```diff
@@ -0,0 +1,369 @@
...
+                          <Typography>£1000</Typography>
+                        </Grid>
+                      </Grid>
+                      <Divider />
+                      <Grid
+                        container
+                        display="flex"
+                        flexDirection="row"
+                        alignItems="center"
+                        sx={{ padding: 2 }}
```
Comment (piotrostr) [inline components/NewBid.tsx:290]: general: you could use the property shorthands, instead of `padding` - `p` or rather than `marginLeft` just `ml`

Comment (rpollock03) [reply components/NewBid.tsx:290]: thanks I can do that going forward!

**Comment by rpollock03** [comment]
@piotrostr should be good to merge now if you can lift block please :)

**Review by piotrostr — APPROVED** [review]

File: components/NewBid.tsx [hunk 3]
```diff
@@ -0,0 +1,369 @@
...
+
+const Heading = styled(Typography)({
+  color: "#101828",
+  fontSize: "1.125rem",
+  lineHeight: "1.75rem",
+  fontWeight: 500,
+})
+
+const Buttonselect = styled(ToggleButton)`
+  textdecoration: "none";
```
[the remark this answers is not in the document]
Comment (rpollock03) [reply components/NewBid.tsx]: You're right. Actually this component wasn't needed so I have removed altogether. Thanks.

**Pull request merged**